perf_test

```diff
@@ -152,41 +151,41 @@ if __name__ == "__main__":
             for bs in args.bandwidth:
                 if args.write:
                     print(f"{done:.1f}%", end="\r")
-                    res = run_fio(fio, bs=bs, chunk_size=chunk_size,
-                                  thread_cnt=thread_cnt,
-                                  cache_size=cache_size, bw=True,
-                                  write=True)
+                    res = run_fio(fio, bs=bs, bw=True, write=True)
+                    add_setup_params(res, chunk_size=chunk_size,
+                                     thread_cnt=thread_cnt,
+                                     cache_size=cache_size)
                     results['BW']['write'].setdefault(bs, [])
                     results['BW']['write'][bs].append(res)
                     done += step
 
                 if args.read:
                     print(f"{done:.1f}%", end="\r")
-                    res = run_fio(fio, bs=bs, chunk_size=chunk_size,
-                                  thread_cnt=thread_cnt,
-                                  cache_size=cache_size, bw=True,
-                                  write=False)
+                    res = run_fio(fio, bs=bs, bw=True, write=False)
+                    add_setup_params(res, chunk_size=chunk_size,
```

Comment:
can we open code this, res.update(dict(chunck_size=...)) if you want to keep the kwargs